project/src/main/puzzle/puzzle-score.gd

```diff
@@ -152,6 +152,7 @@ func make_player_lose() -> void:
 func end_game() -> void:
 	game_active = false
 	game_ended = true
+	no_more_customers = true
```

Comment:
Add a comment describing why we set no_more_customers, because it's a little strange

Reply:
Moved this logic up into the `end_combo()` function where it's more self-documenting. A comment is no longer required.